### PR TITLE
Fix misleading mkpkg warning and docs for integrationtest

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**

### CODE REVIEW

File: mkpkg [hunk 1]
```diff
@@ -264,7 +264,8 @@ class Package:
                 break
         else:
             warnf('Your ARGS list looks weird, maybe you mistakenly used '
-                    'ARGS=list("string")? If so, please use ARGS=["string"] '
+                    'ARGS.extend("string")? If so, please use '
+                    'ARGS.extend(["string") (or just ARGS.append("string")) '
```
Comment (daniel-zullo) [inline mkpkg]: there is a missing closing bracket in `ARGS.extend(["string")`

File: README.rst [hunk 2]
```diff
@@ -1016,11 +1017,12 @@ Integration tests
 ~~~~~~~~~~~~~~~~~
 
 Integration tests are expected to live in the ``$(INTEGRATIONTEST)`` directory
-(``test`` by default), and it is expected that each subdirectory there is
+(``integrationtest`` by default), and it is expected that each
+subdirectory there is
```
Comment (daniel-zullo) [inline README.rst]: you might want to fix this line (merge it to the line above and below)

**Comment by leandro-lucarella-sociomantic** [comment]
Updated

**Review by daniel-zullo — COMMENTED** [review]
It looks to me the test directory in unit tests needs to be update as well.
https://github.com/leandro-lucarella-sociomantic/makd/blob/45d63c217e2286e3cb0835dde78587a1a3d6f225/README.rst#unit-tests

Other than that LGTM.

**Comment by leandro-lucarella-sociomantic** [comment]
Added a few more commits to deal with #103.

**Comment by leandro-lucarella-sociomantic** [comment]
And updated again to fix a comment from @daniel-zullo-sociomantic I missed.

File: README.rst [hunk 3]
```diff
@@ -1016,11 +1017,11 @@ Integration tests
 ~~~~~~~~~~~~~~~~~
 
 Integration tests are expected to live in the ``$(INTEGRATIONTEST)`` directory
-(``test`` by default), and it is expected that each subdirectory there is
-a separate test program, with a ``main.d`` file as the entry point. So the
-typical layout for the ``$(INTEGRATIONTEST)/`` directory is::
+(``integrationtest`` by default), and it is expected that each subdirectory
```
Comment (daniel-zullo) [inline README.rst]: I think there is an `in` missing in this sentence. It should be `, and it is expected that in each subdirectory there is a ...`.
Anyway this suggestion might not be related to the current changes.

**Review by daniel-zullo — APPROVED** [review]
LGTM

File: README.rst (hunk 3 above)
Comment (leandro-lucarella-sociomantic) [reply README.rst]: Fixed anyway.

**Review by daniel-zullo — APPROVED** [review]
LGTM :+1:

**Comment by leandro-lucarella-sociomantic** [comment]
@mathias-lang-sociomantic ping!

**Review by mathias-lang-sociomantic — APPROVED** [review]
LGTM

**Pull request merged**